packages/appstore/src/components/transaction-list/completed-transaction/completed-transaction.tsx

```diff
@@ -99,15 +100,15 @@ const NonPendingTransaction = ({ transaction }: TNonPendingTransaction) => {
                         weight='lighter'
                         line_height={is_mobile ? 's' : 'm'}
                     >
-                        {formatActionType(action_type)}
+                        {localize(formatActionType(action_type))}
```

Comment:
I'm not sure localize can work with variables and functions... Correct me if I'm wrong

Reply:
Fixed